src/base-client.ts

```diff
@@ -58,9 +58,8 @@ export function serializeData(data: Record<string, unknown>): URLSearchParams {
     // Objects/arrays, numbers and booleans get stringified
     // Slack API accepts JSON-stringified-and-url-encoded payloads for objects/arrays
     // Inspired by https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/WebClient.ts#L452
-    const serializedValue: string = (typeof value !== "string"
-      ? JSON.stringify(value)
-      : value);
+    const serializedValue: string =
```

Comment:
Seems like deno fmt changed in `1.21.0` - which is what our GH Action is using. I updated to 1.21.0 locally to run fmt locally, but not sure if we'll want to do that, or peg out GH action to use the previous deno version of `1.20.6`.